beacon_node/src/cli.rs

```diff
@@ -503,7 +503,14 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .requires("execution-endpoint")
                 .takes_value(true)
         )
-
+        .arg(
+            Arg::with_name("execution-timeout-multiplier")
+                .long("execution-timeout-multiplier")
+                .value_name("EXECUTION-TIMEOUT-MULTIPLIER")
```

Comment:
Usually we try to keep these placeholder names as short as possible

```suggestion
                .value_name("NUM")
```